src/Markup/Xamarin.CommunityToolkit.Markup/FuncMultiConverter.cs

```diff
@@ -47,7 +47,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 			return BindableProperty.UnsetValue;
 		}
 
-		public object[] ConvertBack(object value, Type[] targetTypes, object parameter, CultureInfo culture)
+		public object?[]? ConvertBack(object? value, Type[] targetTypes, object? parameter, CultureInfo culture)
```

Comment:
Shouldn't this just be a `object[]?`?

Reply:
You mean that this array shouldn't contain any null values?

Reply:
I'm just wondering if that notation is correct, I didn't see any other place in our code that has two `?` for that notation

Reply:
Yes, it means that array itself could be null and it can contain null items. 

You can think about it like this: `object?` is a nullable object. Array of nullable objects is `object?[]`. But what if array itself could be null? This is how we get `object?[]?`

Reply:
Correct! 

`object[]?` is an array of type `object` can be `null`, but it cannot contain `null` objects

`object?[]` is an array of type `object?` cannot be null, but it can contain `null` objects

`object?[]?` is an array of type `object?` that can itself be null. AND it can contain `null` objects